Fineo/Common/Fineo.Interfaces/IFileStorage.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fineo.Interfaces
+{
+    public interface IFileStorageParams
+    {
+        Dictionary<string, string> Parameters
+        {
+            get;
+            set;
+        }
+    }
+
+    public interface IFileStorage
```

Comment:
Need a methods for:
- get info for single file
- get list of files/subdirs in folder